content/ch-quantum-hardware/Jaynes-Cummings-model.ipynb

```diff
@@ -66,36 +66,8 @@
     "from qiskit.compiler import transpile, assemble\n",
```

Comment:
<p>Because this a standard T1 experiment, you don't need to full-scratch the T1 experiment at pulse level. If the purpose of this section is to write everything with pulse, then this approach should be fine. Otherwise, you can use</p><pre class="ql-syntax" spellcheck="false">qc = QuantumCircuit(1, 1)
qc.x(0)
qc.delay(delay, 0)
qc.measure(0, 0)
</pre><p>and transpile it with initial layout.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>